src/style.scss

```diff
@@ -16,3 +16,7 @@
     opacity: 1;
   }
 }
+
+a[id*="_ccLogo"] {
```

Comment:
can you move it please within the `.datavisyn-anatomogram` namespace to avoid collisions with other plugins.